setup.py

```diff
@@ -38,19 +38,18 @@
         'Programming Language :: Python :: 3.6',
     ],
     install_requires=[
-        'requests==2.22.0',
-        'certifi==2019.11.28',
-        'urllib3==1.25.7',
-        'chardet==3.0.4',
-        'idna==2.8',
-        'coverage==4.4',
-        'pytutor==1.0.0',
-        'ast-scope==0.3.1',
-        'attrs==19.3.0',
-        'pyaes==1.6.1',
-        'colorama==0.4.3',
-        'display-timedelta==1.1',
-        'filelock==3.0.12',
-        'lark==0.12.0',
+        'requests>=2.22.0',
+        'certifi>=2019.11.28',
+        'urllib3>=1.25.7',
+        'chardet>=3.0.4',
+        'idna>=2.8',
+        'coverage>=4.4',
+        'pytutor>=1.0.0',
+        'ast-scope>=0.3.1',
+        'attrs>=19.3.0',
+        'pyaes>=1.6.1',
+        'colorama>=0.4.3',
+        'display-timedelta>=1.1',
+        'filelock>=3.0.12',
```

Comment:
this seems to be missing the `lark` import, which is necessary for 61a assignments -- please add it back!